components/RichText/Quote.module.scss

```diff
@@ -18,16 +18,15 @@ $content-padding: $spacing-3 + $spacing-4;
 .content {
     @include paragraph;
 
-    background: $color-base-50;
     padding: $spacing-4;
     position: relative;
 
     &::before {
         content: "";
         position: absolute;
         top: 50%;
-        background-color: $color-base-400;
-        width: 2px;
+        background-color: var(--prezly-border-color-secondary);
+        width: 4px;
```

Comment:
Checking we meant to increase the quote border width.

Reply:
But I did notice that I forgot to add the background color to the Quote.

Reply:
Wait, I didn't actually, because in Figma, quote is transparent whereas in production it's using something that would be a "bg secondary" in the new variables.. so not sure which one to use.

Reply:
So either we go with the new transparent + 4px width or keep 2px width but use the secondary bg color.

Reply:
hmm - I wonder if quotes can be included in anything? I think it's OK to use transparent TBH as they should be always on something with a background color.

Reply:
Quote width must be from Hugo's new designs - I'm happy w that then. No further comments.